app/Providers/GitHubServiceProvider.php

```diff
@@ -12,7 +12,7 @@ public function register()
     {
         $this->app->bind(GitHubClient::class, function ($app) {
             $client = new GitHubClient(null, 'squirrel-girl-preview');
-            $client->authenticate(config('services.github.client_id'), config('services.github.client_secret'), GitHubClient::AUTH_URL_CLIENT_ID);
```

Comment:
I think we still need `client_secret` as the 2nd parameter/"password", so basic auth would use the format `<client_id>:<client_secret>` to authenticate. Can you confirm this works as-is?

Reply:
@sbine interestingly, that worked as is (I was testing this running `php artisan sync:repo`).

I just pushed a commit that re-adds the secret and tested that it worked with the same command as above. It's probably best to include the secret in case there we make API requests in the future that require it.